conf/app.ini

```diff
@@ -46,6 +46,10 @@ DOMAIN = localhost
 ROOT_URL = %(PROTOCOL)s://%(DOMAIN)s:%(HTTP_PORT)s/
 HTTP_ADDR =
 HTTP_PORT = 3000
+; Local (DMZ) URL for gogs workers (such as ssh update) accessing web service.
+; In most cases you may leave it empty. Defaults: http://localhost:HTTP_PORT/
+; You may need to set it only if ssh server node is not the same as http node.
+LOCAL_ROOT_URL =
```

Comment:
I didn't want to on purpose. It is better to not expose LOCAL_ROOT_URL in `custom/conf/app.ini` at all as for most users this would be irrelevant.
Doing this as you suggest we risk some rookie user changing `HTTP_PORT` but forgetting about port in `LOCAL_ROOT_URL` :)